src/main/resources/application.properties

```
@@ -1,5 +1,5 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/book_store
-spring.datasource.username=root
+spring.datasource.url=jdbc:mysqldb://localhost:3305/book_store
```

```suggestion
spring.datasource.url=jdbc:mysql://mysqldb:3305/book_store
```
mysql is type of DB you use and mysqldb is part of url (name of container in docker)